tests/auth_tests/test_forms.py

```diff
@@ -740,6 +740,19 @@ def test_username_field_autocapitalize_none(self):
         form = UserChangeForm()
         self.assertEqual(form.fields['username'].widget.attrs.get('autocapitalize'), 'none')
 
+    def test_only_superuser_should_be_able_to_grant_superuser(self):
```

Comment:
Should this test case be split into two test cases?

Reply:
It can. But I thought it will be easier to read and maintain this way.